BeatSaverVoting/manifest.json

```diff
@@ -2,14 +2,14 @@
   "$schema": "https://raw.githubusercontent.com/nike4613/ModSaber-MetadataFileSchema/master/Schema.json",
   "author": "Top_Cat",
   "description": "A plugin for up/downvoting songs on BeatSaver",
-  "gameVersion": "1.34.0",
+  "gameVersion": "1.36.0",
   "id": "BeatSaverVoting",
   "name": "BeatSaverVoting",
   "dependsOn": {
     "BSIPA": "^4.2.0",
-    "BeatSaberMarkupLanguage": "^1.5.5",
-    "BS Utils": "^1.11.0",
-    "SongCore": "^3.7.0"
+    "BeatSaberMarkupLanguage": "^1.9.0",
+    "BS Utils": "^1.12.0",
+    "SongCore": "^3.13.0"
```

Comment:
Are there new BSML features / BS Utils hooks we need or are you just upgrading these for fun?

Reply:
The dependency version bump is to force the mod to not load on older versions of the game. 

The dependency version I chose is the ones for 1.34.4

Well I guess the SongCore version bump is needed anyways because of the type changes. I can revert the others if you prefer.